docs/source/grfn_api_v1.0.0.yaml

```diff
@@ -0,0 +1,211 @@
+swagger: "2.0"
+info:
+  description: "This document describes the API for interacting with AutoMATES GrFN."
+  version: "1.0.0"
+  title: "AutoMATES GrFN"
+  termsOfService: ""
+  contact:
+    email: "claytonm@arizona.edu"
+  license:
+    name: "TODO Apache 2.0"
+    url: "http://www.apache.org/licenses/LICENSE-2.0.html"
+host: "TODOhost"
+basePath: "/v1"
+tags:
+- name: "generate"
```

Comment:
@dpdicken I think it would be better to name this endpoint `/translate` since that implies a GrFN translation from source input.